internal/evaluator/conftest_evaluator.go

```diff
@@ -255,3 +280,86 @@ func isResultEffective(failure output.Result, now time.Time) bool {
 	}
 	return effectiveOn.Before(now)
 }
+
+// isResultIncluded returns whether or not the result should be included or
+// discarded based on the policy configuration.
+func (c conftestEvaluator) isResultIncluded(result output.Result) bool {
+	matchers := makeMatchers(extractCode(result))
+	includes := []string{"*"}
+	excludes := []string{}
+	if c.policy.Configuration != nil {
+		// If collections is not empty, then only include rules that mention
+		// the collection.
+		if len(c.policy.Configuration.Collections) > 0 {
+			collections := extractCollections(result)
+			if !hasAnyMatch(collections, c.policy.Configuration.Collections) {
+				return false
+			}
+		}
+		if len(c.policy.Configuration.Include) > 0 {
+			includes = c.policy.Configuration.Include
+		}
+		if len(c.policy.Configuration.Exclude) > 0 {
+			excludes = c.policy.Configuration.Exclude
+		}
+	}
+
+	if c.policy.Exceptions != nil {
+		// TODO: NonBlocking is deprecated. Remove it eventually
+		excludes = append(excludes, c.policy.Exceptions.NonBlocking...)
+	}
+	return hasAnyMatch(matchers, includes) && !hasAnyMatch(matchers, excludes)
+}
+
+// hasAnyMatch returns true if the haystack contains any of the needles.
+func hasAnyMatch(needles, haystack []string) bool {
+	for _, needle := range needles {
+		for _, hay := range haystack {
+			if needle == hay {
```

Comment:
I did a google and found [slice.Contains](https://pkg.go.dev/golang.org/x/exp/slices#Contains) - is it any use here?

Reply:
Sweet! I missed that. Updated.